sdk/purview/Azure.ResourceManager.Purview/tests/Scenario/PurviewAccountTests.cs

```diff
@@ -0,0 +1,112 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Purview.Tests;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Purview.Samples.Scenario
+{
+    internal class PurviewAccountTests : PurviewManagementTestBase
+    {
+        private const string _purviewAccountPrefix = "purviewaccount";
+        private PurviewAccountCollection _purviewAccountCollection;
+
+        public PurviewAccountTests(bool isAsync) : base(isAsync)//, RecordedTestMode.Record)
+        {
+        }
+
+        [SetUp]
+        public async Task TestSetUp()
+        {
+            var rg = await CreateResourceGroup();
+            _purviewAccountCollection = rg.GetPurviewAccounts();
+        }
+
+        private async Task<PurviewAccountResource> CreatePurviewAccount(string accountName)
+        {
+            PurviewAccountData data = new PurviewAccountData(DefaultLocation)
+            {
+                Identity = new ManagedServiceIdentity(ManagedServiceIdentityType.SystemAssigned)
+            };
+            var purviewAccountLro = await _purviewAccountCollection.CreateOrUpdateAsync(WaitUntil.Completed, accountName, data);
+
+            // Wait 180 seconds for service deployment to complete.
+            if (Mode == RecordedTestMode.Record)
```

Comment:
Why need to wait? The `Create`  method should return until the resource has been created

Reply:
If we want to update or delete it, it will return 409 (Conflict): `Account PurviewAccount cannot be updated when it is in Creating state`